css/index.css

```diff
@@ -1,3 +1,324 @@
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+
+
+.container {
+	max-width: 1024px;
+    margin: 0 auto;
+    
+}
+
+button {
+	background: inherit;
+	font-family: inherit;
+	border: 1px solid black;
+	height: 30px;
+	width: 150px;
+}
+
+/* styling my header section */
+
+.index .banner {
+	display: flex;
+	justify-content: space-around;
+    align-items: center;
+    display: flex;
+    flex-wrap: wrap;
+    /* justify-content: space-between; */
+    /* flex-direction: column-reverse; */
+   
+}
+
+/* BANNER MEDIA-Q */
+
+  @media(max-width: 800px) {
+    .banner {
+      flex-direction: column;
+      flex-direction: column-reverse;
+    }
+  }
+
+  @media(max-width: 800px) {
+    .banner .logo {
+        display: flex;
+      flex-direction: column;
+      flex-direction: column-reverse;
+      margin: 5%;
+    }
+  }
+
+
+.index header .banner nav a {
+	text-decoration: none;
+	font-family: inherit;
+    color: lightgrey;
+    margin: 30px;
+}
+
+@media(max-width: 800px) { 
+.index header .banner nav a{
+    margin: 0 auto;
+}
+}
+
+/* BANNER NAV MEDIA-Q */
+
+  @media(max-width: 515px) {
+    .index header .banner nav a {
+        display: flex;
+        justify-content: center;
+        margin: 0 auto;
+    }
+  }
+
+
+/* NAV BORDER MEDIA Q */
+  @media(max-width: 515px) {
+    .index header .banner nav a {
+        border: 0.2px solid rgb(133, 129, 129);
+        width: 600px;
+    }
+  }
+
+  @media(max-width: 800px) {
+    .index header .banner nav a {
+        /* border: 0.2px solid rgb(133, 129, 129); */
+        padding: 10px;
+        width: 600px;
```

Comment:
I'd increase this padding to 20px for bigger buttons, better for thumbs on phones